src/dr_client.py

```diff
@@ -519,15 +522,25 @@ def set_registered_model_global(self, registered_model_name, is_global):
                 f"Failed to find registered model by name: {registered_model_name}"
             )
 
-        if registered_model.get("isGlobal", None) == is_global:
+        payload = {}
+
+        if description is not None and description != registered_model["description"]:
+            payload["description"] = description
+
+        if is_global is not None and is_global != registered_model.get("isGlobal", None):
```

Comment:
I noticed that the default value for `is_global` is `False`. So, if it does not exists, you should consider it as False.

Reply:
Same here. I'm ignoring the value if it is not present like it is done in other places in the code